sdcm/cluster_k8s/__init__.py

```diff
@@ -300,6 +300,17 @@ def __init__(self, params: dict, user_prefix: str = '', region_name: str = None,
     _scylla_operator_namespace = SCYLLA_OPERATOR_NAMESPACE
     _scylla_manager_namespace = SCYLLA_MANAGER_NAMESPACE
 
+    @cached_property
+    def tenants_number(self) -> int:
+        for stress_param in self.params.stress_cmd_params:
+            stress_cmds = self.params.get(stress_param)
+            if not isinstance(stress_cmds, list):
+                continue
+            if any((isinstance(stress_cmd, list) for stress_cmd in stress_cmds)):
+                if (tenants_number := len(stress_cmds)) > 1:
```

Comment:
So this is instead of adding a tenant number configuration ? I think being explicit in the configuration is a bit more clear than this

Reply:
Cluster number directly depends on the stress commands structure. So, it is question of depending on one kind of options or two. I don't see any need in updating 2 places instead of one when we want to change number of clusters.